src/include/sof/lib/alloc.h

```diff
@@ -24,6 +24,9 @@
 #include <stddef.h>
 #include <stdint.h>
 
+#include <sof/lib/cache.h>
+#include <sof/lib/memory.h>
+
```

Comment:
Won't approve until you have actually checked that these are indeed needed and updated commit message to no longer state "TODO check."

Reply:
you are right - this one has not been checked.

Reply:
......and it's not needed now. Will remove.